src/main/java/me/pesekjak/machine/utils/FileUtils.java

```diff
@@ -13,7 +13,8 @@ public final class FileUtils {
             .getProtectionDomain()
             .getCodeSource()
             .getLocation()
-            .getFile());
+            .getFile()
+            .replaceAll("%20", " "));
```

Comment:
You should use `String.replace(String, String)` instead of `String.replaceAll(String, String)` because `String.replaceAll` uses regex and you're not using regex so you can understand the intentions better.

Reply:
Thanks for your review, me and the team have decided to get to the root of the problem since space isn't the only character affected.